internal/langserver/handlers/execute_command_rootmodules_test.go

```diff
@@ -169,15 +170,18 @@ func TestLangServer_workspaceExecuteCommand_rootmodules_multiple(t *testing.T) {
 			"doneLoading": true,
 			"rootModules": [
 				{
-					"uri": %q
+					"uri": %q,
+					"name": %q
 				},
 				{
-					"uri": %q
+					"uri": %q,
+					"name": %q
 				},
 				{
-					"uri": %q
+					"uri": %q,
+					"name": %q
 				}
 			]
 		}
-	}`, dev.URI(), prod.URI(), staging.URI()))
+	}`, dev.URI(), "env/dev", prod.URI(), "env/prod", staging.URI(), "env/staging"))
```

Comment:
I think you might need to use `filepath.Join` here for cross-platform compatibility, as shown by the failing Windows test.